index.bs

```diff
@@ -7903,13 +7907,21 @@ Constructors</h4>
 <dl dfn-type=constructor dfn-for="MediaStreamAudioSourceNode">
 	: <dfn>MediaStreamAudioSourceNode(context, options)</dfn>
 	::
-		Let <var>node</var> be a new {{MediaStreamAudioSourceNode}}
-		object. <a href="#audionode-constructor-init">Initialize</a>
-		<var>node</var>, and return <var>node</var>.
-
+			1. If <var>context</var> is not an {{AudioContext}}, throw an
```

Comment:
An editorial nit: why this line is indented deeper?

Reply:
I don't know, I get an error from bikeshed if I don't do this, so I did it.

> +FATAL ERROR: Line 7710 isn't indented enough (needs 2 indents) to be valid Markdown:
> ...